src/tutorial/microbe_stage/OrganelleDivisionTutorial.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace Tutorial
+{
+    using System;
+
+    /// <summary>
+    ///   Tells the player about the day and night cycle
+    /// </summary>
+    public class OrganelleDivisionTutorial : TutorialPhase
+    {
+        public override string ClosedByName => "OrganelleDivisionTutorial";
+
+        public override void ApplyGUIState(MicrobeTutorialGUI gui)
+        {
+            gui.OrganelleDivisionTutorialVisible = ShownCurrently;
+        }
+
+        public override bool CheckEvent(TutorialState overallState, TutorialEventType eventType, EventArgs args,
+            object sender)
+        {
+            switch (eventType)
+            {
+                case TutorialEventType.MicrobeNonCytoplasmOrganelleDivided:
+                {
+                    if (!HasBeenShown && CanTrigger && !overallState.TutorialActive())
+                    {
+                        Show();
```

Comment:
I think this tutorial should have an end trigger to automatically close after 60 seconds even if the player doesn't close this explicitly.